src/main/resources/locale_fr_FR.properties

```
@@ -0,0 +1,207 @@
+command.gdpr.description=Requiert une copie des données ou leur suppression.
+command.gdpr.sub.delete=Requête la suppression de tes données.
+command.gdpr.sub.delete.received=Ta suppression de données est programmée. Elles seront supprimées dans les prochaines heures.
+command.gdpr.sub.delete.scheduled=Tes données sont déjà programmées pour être supprimées. Elles seront supprimées dans les prochaines heures.
+command.gdpr.sub.request=Requête une copie de tes données.
+command.gdpr.sub.request.received=Nous avons reçu ta demande. Tu recevras tes données dans les prochaines heures. Tu peux demander tes données tous les 30 jours.
+command.gdpr.sub.request.requested=Tu as demandé tes données au cours des 30 derniers jours. Tu ne peux demander tes données que tous les 30 jours.
+command.help.alias=Alias
+command.help.description=Demande de l'aide !
+command.help.list.list=Voici une liste de toutes les commandes.\nUtilise %COMMAND% pour plus d'informations :
+command.help.list.title=Reputation Bot Aide
+command.help.subCommands=Sous-commandes
+command.help.title=%COMMAND% - Aide
+command.help.usage=Utilisation
+command.info.art=Travaux d'art et image de marque
+command.info.contributor=Contributeur
+command.info.description=Recevoir des informations sur moi !
+command.info.embedTitle=ReputationBot
+command.info.links=[Invite-moi](%INVITE%) ᠅ [Support](%SUPPORT%) ᠅ [TOS & Privacy](%TOS%) ᠅ [Website](%WEBSITE%)
+command.info.source=Code source
+command.info.version=Version
+command.invite.click=[Cliquez ici](%URL%)
+command.invite.description=Invite le bot sur ton serveur.
+command.invite.title=Invite-moi
+command.locale.current=La langue actuelle est %LOCALE%.
+command.locale.description=Recevoir la langue actuelle
+command.locale.error.invalidLocale=Cette langue n'existe pas.
+command.locale.sub.list=Recevoir toutes les langues disponibles
+command.locale.sub.list.list=Les langues actuellement disponibles :
+command.locale.sub.set=Définir la langue
+command.locale.sub.set.set=La langue est réglée sur %LOCALE%.
+command.log.description=Obtenir des informations sur la réputation d'un utilisateur ou d'un message.
+command.log.donatedLog=Réputation des dons de l'utilisateur %USER%.
+command.log.messageLog=Réputations reçues pour le message %ID%.
+command.log.receivedLog=Réception des réputations de l'utilisateur %USER%.
+command.log.sub.donated=Recevoir la réputation donnée d'un utilisateur.
+command.log.sub.message=Recevoir le journal d'un message.
+command.log.sub.received=Obtient la réputation reçue d'un utilisateur.
+command.prefix.changed=Préfixe défini sur %PREFIX%.
+command.prefix.description=Montrer le préfixe
+command.prefix.show=Le préfixe actuel est %PREFIX%.
+command.prefix.sub.reset=Réinitialise le préfixe
+command.prefix.sub.set=Définir le préfixe
+command.prune.description=Élimine les utilisateurs non valides ou un utilisateur spécifique.
+command.prune.sub.guild=Remove reptutation of users which are no longer on the guild.
```

Looks untranslated

DeepL suggestion:
```suggestion
command.prune.sub.guild=Supprimer la reptation des utilisateurs qui ne sont plus sur la guilde.
```